src/libmusig2.h

```diff
@@ -118,25 +118,26 @@ int musig2_aggregate_R(musig2_context_sig *mcs, secp256k1_pubkey *batch_list);
 
 /** Function    : musig2_sign
  *  Purpose     : Starts the signature process for signer and calls `musig2_sign_partial`.
- *                Returns 1 if partial signature is created successfully, 0 otherwise.
+ *                Returns 1 if partial signature is created successfully, -1 if the corresponding commitment is NULL, 0 otherwise.
```

Comment:
I updated the function documentation as they include all other return values described.